OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -10171,7 +10171,7 @@ protected function getInitialAttributeHelper
   input list<DAE.ComponentRef> iterationVars "list of iterationvars from InitializationDAE";
   output SimCodeVar.Initial initial_;
 algorithm
-  if isFixed and startValueIsConstOrNone(var) then
+  if not isFixed and startValueIsConstOrNone(var) then
```

Comment:
@mahge We did this for some purpose but i don't remember it now, I have to look into the specification

Reply:
It was added by #568. I was going to talk to you about it. I just wanted to see what it does to the tests. We can have a meeting and figure this out because something does not look right here. This is removing start values that were given by the Modelica code itself for normal (non-fixed) variables (not parameters). Maybe it is what should be done but then we need to figure out to make sure those start values are used where needed by the generated code.